main.css

```diff
@@ -94,15 +94,56 @@ footer
     background-color: #061428;
 }
 
-#sidebar
+#mobileMenuIcon
 {
     position: fixed;
+    top: 35px;
+    left: 5px;
+    width: 48px;
+    height: 48px;
+    z-index: 9999;
+    background: url("assets/mobile-ui/mobilemenu.png");
+}
+
+#mobileHideMenu
```

Comment:
There's no explicit button as I'm using the grayed out area as the return button for now, but will do.